ENIGMAsystem/SHELL/Collision_Systems/Precise/Info/About.ey

```diff
@@ -8,5 +8,4 @@ Description: Collisions register using sprite image sources.
 Author: forthevin
 
 Represents:
-	Build-platforms: All
-
+	Build-platforms: All
```

Comment:
Why did you delete the newline from the end of this file?

Reply:
OK I don't remember this but fixed this in the new commits